From: Daniel Baker <dbaker@mozilla.com>
Date: Tue, 27 Aug 2024 18:37:13 -0600
Subject: (tmp-cherry-pick) Revert "Add a dependancy on
 //third_party/jni_zero:jni_zero_java" (83671efdb9)

This reverts commit 5ccec98826adc84bdb4c8177580a6f23074d8058.

Reason for revert: Breaks downstream project.

Original change's description:
> Add a dependancy on //third_party/jni_zero:jni_zero_java
>
> JniZero's native side expects JniZero's java side to exist, add a
> dependancy on //third_party/jni_zero:jni_zero_java from
> sdk/android:base_java.
>
> Bug: webrtc:353174456
> Change-Id: I9240eab3da78efd394e4ee581cf71fd41976eb48
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/357200
> Commit-Queue: Mohamed Heikal <mheikal@google.com>
> Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
> Auto-Submit: Mohamed Heikal <mheikal@google.com>
> Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#42639}

Bug: webrtc:353174456
Change-Id: I58bdb9e1797f6d2f716225abc5c337cd90290159
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/357460
Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
Reviewed-by: Harald Alvestrand <hta@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Reviewed-by: Florent Castelli <orphis@webrtc.org>
Reviewed-by: Mohamed Heikal <mheikal@google.com>
Cr-Commit-Position: refs/heads/main@{#42647}
---
 sdk/android/BUILD.gn | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sdk/android/BUILD.gn b/sdk/android/BUILD.gn
index 641444b301..be84a6158b 100644
--- a/sdk/android/BUILD.gn
+++ b/sdk/android/BUILD.gn
@@ -172,7 +172,6 @@ if (is_android) {
     deps = [
       "//rtc_base:base_java",
       "//third_party/androidx:androidx_annotation_annotation_java",
-      "//third_party/jni_zero:jni_zero_java",
     ]
   }
 
